demisto_sdk/commands/lint/linter.py

```diff
@@ -403,6 +405,8 @@ def _run_xsoar_linter(self, py_num: float, lint_files: List[Path]) -> Tuple[int,
                 myenv['LONGRUNNING'] = 'True'
             if py_num < 3:
                 myenv['PY2'] = 'True'
+            myenv['commands'] = ','.join([str(elem) for elem in self._facts['commands']]) \
+                if self._facts['commands'] else ''
             stdout, stderr, exit_code = run_command_os(
                 command=build_xsoar_linter_command(lint_files, py_num, self._facts.get('support_level', 'base')),
```

Comment:
can't the `commands` be passed here?

Reply:
No , xsoar linter runs as a pylint plugin , each support level is a standalone plugin.
Pylints plugins are running inside the pylint code, this is the reason why we cant use our constants and fact inside it.